app/main.js

```diff
@@ -18,17 +18,37 @@ commander.program
   .option('-a, --all', 'Scrape all airports in all the countries')
   .option('-p, --prod', 'Run in production mode')
   .option('-cm, --completed-map', 'World map link of implemented playbooks')
+  .option('-ld,--list-disabled', 'List out all disabled playbooks')
   .parse(process.argv);
 
 const options = commander.program.opts();
 
+//Function to list out all disabled playbooks (enable: false)
+const listDisabled = (playbooksDir) => {};
+
 async function main() {
   logger.debug(process.argv.join(' '), { type: 'general' });
 
   const playbooksDir = `${process.cwd()}/playbooks`;
 
   let prodMode = false;
 
+  //List disabled playbooks
```

Comment:
Fix the linting on the comment. You can use `npm run lint`and find what is going wrong.

Reply:
Hi. Last time I fixed that issue with new commit and somehow it did not go through to this pull request. I made another one. Can you please check @drph4nt0m ? Thanks